tests/page_test.go

```diff
@@ -960,7 +960,7 @@ func TestPageOn(t *testing.T) { //nolint:gocognit
 			assertFn: func(t *testing.T, cm *common.ConsoleMessage) {
 				t.Helper()
 				assert.Equal(t, "timeEnd", cm.Type)
-				assert.Regexp(t, "^k6: [0-9]*.[0-9]*", cm.Text, `expected prefix "k6: 0." but got %q`, cm.Text)
+				assert.Regexp(t, "^k6: [0-9]*.[0-9]*", cm.Text, `expected prefix "k6: <a float>" but got %q`, cm.Text)
```

Comment:
I think this should be like:

```suggestion
				assert.Regexp(t, "^k6: [0-9]*\.[0-9]*", cm.Text, `expected prefix "k6: <a float>" but got %q`, cm.Text)
```

Otherwise `.` will match to any character.

Reply:
If we always expect a number `*`s should be `+`s instead. Like this.

```suggestion
				assert.Regexp(t, "^k6: [0-9]+\.[0-9]+", cm.Text, `expected prefix "k6: <a float>" but got %q`, cm.Text)
```

Reply:
Resolved in 138f3b8ac8cb3d844d357e6ac57248a7b9637340

Reply:
Thanks, should we mind [this](https://github.com/grafana/xk6-browser/pull/1139#discussion_r1428000647)?

Reply:
`*` and `+` work for this use case.

Reply:
OK, so, we expect n.m, .m, or n. then. Alright 👍

Reply:
Ah, wait, i see resolved in 559a597015f757602f77eea4d79c9fd0566b497d

Reply:
Regexes are always P*ITA :)